lib/graph-service-worker.js

```diff
@@ -114,9 +114,9 @@ function find (rootname, arr, done) {
 function fileEnv (state) {
   var script = [
     'https://cdn.polyfill.io/v2/polyfill.min.js',
-    `${state.scripts.bundle.hash}/bundle.js`
+    `${state.scripts.bundle.hash.hexSlice().slice(0, 16)}/bundle.js`
```

Comment:
afaik `hexSlice` is not public API, maybe better to use `.toString('hex', 0, 16)`?

Reply:
Updated to use `.toString()`! Kept the `.slice()` method in there because we had to do `.toString('hex', 0, 8)` which isn't as clear that it returns 16 bytes.